plone/app/portlets/portlets/actions.pt

```diff
@@ -4,15 +4,15 @@
       tal:omit-tag=""
       i18n:domain="plone">
 
-<section class="portlet portletActions" role="menu"
+<div class="portlet portletActions"
```

Comment:
'aside' should be the default option for portlets, as we can have many different portlets and 'aside' is about as generic as it gets. 

However, for specific portlets, like  'portletactions', "menu" is definitely the better choice, as it is indeed a menu of options, and will allow users using Assistive Technology to get there, and to understand the meaning, quicker.